packages/web/src/common/store/smart-collection/smartCollections.ts

```diff
@@ -27,7 +27,7 @@ export const HEAVY_ROTATION: SmartCollection = {
   description: 'Your top tracks, in one place',
   gradient: 'linear-gradient(316deg, #C751C0 0%, #4158D0 100%)',
   shadow: 'rgba(196,81,193,0.35)',
-  icon: IconExploreRotation,
+  icon: IconExploreRotation as any,
```

Comment:
Can't explaining what's up here. I've done a bit of icon work in Harmony to get the types working so maybe I can help?

Reply:
`vite-plugin-svgr` exports the svg as the default export, and the type is slightly different. The `any` isn't needed if we were only running web BUT: This file gets imported by mobile and the types break in mobile. Once we complete the common migration this won't be an issue (we can define the canonical svg type in common)